mlflow/server/js/src/experiment-tracking/components/MetricView.css

```diff
@@ -30,7 +30,6 @@ div.MetricView h1 {
 .metrics-plot-container .plot-controls {
   display: flex;
   flex-direction: column;
-  justify-content: center;
```

Comment:
I remember this is put here to handle some of the alignment issues when we have a line chart. It may not be an issue anymore but could you please try a run with metrics history and make sure the controls and the line chart aligns normally? I remember @dbczumar have a script to generate test metrics histories.

Reply:
I ran the script to generate test metrics histories and played with the metric view UI and did not find any alignment problem. Not sure what case will trigger it but it looks good for now.

![Screen Shot 2020-05-11 at 2 42 06 PM](https://user-images.githubusercontent.com/64800904/81615396-5f366800-9396-11ea-9eef-0727c327d7c0.png)
![Screen Shot 2020-05-11 at 2 42 52 PM](https://user-images.githubusercontent.com/64800904/81615402-6198c200-9396-11ea-9693-b2f054691235.png)
![Screen Shot 2020-05-11 at 2 44 17 PM](https://user-images.githubusercontent.com/64800904/81615405-62315880-9396-11ea-8f91-c8f788e37c36.png)
![Screen Shot 2020-05-11 at 2 45 40 PM](https://user-images.githubusercontent.com/64800904/81615407-62c9ef00-9396-11ea-8ffe-4ebb9013a862.png)